docs/spec_draft.md

```diff
@@ -1461,6 +1462,89 @@ More formally, `result[i0, ..., iR-1] = operand[j0, ..., jR-1]` where
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.sort
+
+### Semantics
+
+Sorts a variadic number of tensors in `inputs` together, according to a custom 
+`comparator`, along the given `dimension` and produces a variadic number of
+tensors as `results`. If `is_stable` is true, then the sorting is stable, that
+is, relative order of elements considered to be equal by the comparator is
+preserved.
+
+More formally, for all `0 <= id < jd < dim(inputs[0], d)`, either
+`compare_i_j = compare_j_i = false` or `compare_i_j = true`, where:
+  1. `compare_i_j` $=$ `comparator(inputs[0][i], inputs[0][j], inputs[1][i], inputs[1][j], ...)`.
+  1. For all indices `i = [i0, ..., iR-1]` and `j = [j0, ..., jR-1]`.
+  1. Where `i` $=$ `j` everywhere except for the `d`th dimension.
```

Comment:
Shouldn't this be 
id $=$ jd except for d = `dimension` ?
@burmako 
@subhankarshah

Reply:
I & j refer to the entire array of indices, so this says i = j everywhere in the array except for dth dimension

Reply:
I am not sure which `d` are we referring to here. At the top we use `d` in a generic sense: `for all 0 <= id < jd < dim(inputs[0], d)`  to say the index components are in bounds of the dimension of the input tensor.

Reply:
I I missed the explanation just below. Please ignore my comment.